src/utils/frontmatter.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+function parse(text) {
+  let delimiter;
+
+  if (text.indexOf("---") === 0) {
+    delimiter = "---";
+  }
+
+  if (text.indexOf("+++") === 0) {
+    delimiter = "+++";
+  }
+
+  let end = -1;
+
+  if (!delimiter || (end = text.indexOf(`\n${delimiter}`, 3)) === -1) {
```

Comment:
You shouldn’t need the start index — if the file starts with `---`, it by definition doesn’t start with `\n---` and we’ll instead capture the whole document even without the start index.

Reply:
> and we’ll instead capture the whole document even without the start index.

I don't understand what you mean. You're right it doesn't need the start index but that won't change anything